stable/falco/templates/service.yaml

```diff
@@ -15,3 +15,23 @@ spec:
   - protocol: TCP
     port: {{ .Values.falco.webserver.listenPort }}
 {{- end }}
+{{- if .Values.falco.grpc.enabled }}
```

Comment:
Some people splits the file for each service, but I don't have a pretty strong opinion on this.

We can keep like this and if we find some issues in the future, we can split it :)